server/routes/posts.js

```diff
@@ -252,6 +264,18 @@ router.post('/comments/:postID', async (req, res, _next) => {
         })
         comment["author"] = author
 
+
+        const userInteractionData = await prisma.interactionData.findUnique({
+            where: { userID }
+        })
+
+        // Update the user's interaction data
+        const { newAverage, newInteractionCount } = recalculateInteractionAverages(userInteractionData, COMMENT)
+        await prisma.interactionData.update({
+            where: { userID },
+            data: { commentInteractionCount: newInteractionCount, averageCommentInteractionTime: newAverage }
```

Comment:
would it makes sense to include this db call in the `recalculateInteractionAverages` so that all the logic related to update that data is isolated from the rest of the logic? That would also help to avoid calling that db endpoint in all the places where you are recalculating interaction averages

Reply:
Same comment would apply to `recalculateSessionAverages`